lib/artifacts/lambda.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const cf = require('@mapbox/cloudfriend');
 
 const DEFAULT_LAMBDA_SETTINGS = {
-  RUNTIME: 'nodejs6.10',
+  RUNTIME: 'nodejs8.10',
   MEMORY_SIZE: 128,
   TIMEOUT: 60
```

Comment:
Any reason we shouldn't just make the default timeout the max of 900 seconds?

Reply:
Don't see why not.

Reply:
Also the timeout check here needs to be updated https://github.com/mapbox/lambda-cfn/blob/master/lib/artifacts/lambda.js#L52

Reply:
And max memory is 3008 now, so these too https://github.com/mapbox/lambda-cfn/blob/dcbc983625cf01c8d17f647a0066fe0a8fec24a1/lib/artifacts/lambda.js#L64-L68

Reply:
Hm, maybe the argument against defaulting to the highest possible timeout is that, in most cases, you don't want to get billed for 15 minutes of waiting if the Lambda fails 15 seconds in.

Reply:
true, but then I'd say you've got a code problem, not a timeout problem.

Reply:
I'd call the 60 second timeout and 128 memory size 'safe defaults' here.